src/modifiers/index.ts

```diff
@@ -41,3 +41,11 @@ export const Modifiers = [
     Torch,
     Underdog,
 ].sort((a, b) => a.name.localeCompare(b.name));
+
+export const ModifiersTajemniczy = [
```

Comment:
Osobiście przeniósłbym to gdzieś do `src/app/tajemniczy/_modifiers` (i podobnie z rolami). Głowny katalog wskazuje na to, że rzeczy tutaj mogą być użyte wszędzie, a użyjemy ich tylko na stronie tajemniczego - ergo powinny być blisko tej strony.